components/empty_i2c_component/README.md

```diff
@@ -3,9 +3,9 @@
 
 empty_i2c_component:
   id: empty_i2c_component_1
-  address: 0x01  # optional
+  address: 0x01  # optional, default i2c address of device
```

Comment:
This isn't correct.  That *is* the address, not the default.  You're overriding the default by putting the address in the config.
The other changes are correct.